textproc/colout/Portfile

```diff
@@ -19,11 +19,11 @@ long_description    ${name} read lines of text stream on the standard input and
 
 homepage            https://nojhan.github.io/colout/
 
-checksums           rmd160  09e17028c02065efc237ed73dbbc415abde78d4e \
-                    sha256  1dfafc1b62054f08ffdcf813b1207e6a67e2f185fc6df329cea78b7425a18256 \
-                    size    36856
+checksums           rmd160  c39174cfe17941160a90e22f3d4f48f660716703 \
+                    sha256  6d13793207b27a5175592818140a83d748e5604b6d6aa63f8f7865cf1d481eee \
+                    size    52783
 
-python.default_version 310
+python.default_version 311
 
 patchfiles-append   patch-setup.py.diff
```

Comment:
is this patching still needed; can it not use "setuptools_scm"?

Reply:
I did not try, to be honest :)
Usually I keep patches, if they do not look suspicious and do not break the build.

I will try.